apps/SageAccountWeb/src/App.tsx

```diff
@@ -46,154 +48,154 @@ const queryClient = new QueryClient({
 })
 
 function App() {
+  const history = useHistory()
   return (
     <div className="App">
-      <Router>
```

Comment:
Moved this `BrowserRouter` to an outer wrapper so that this component could use the `useHistory` hook